HW12/SQLAlchemy/linguist.py

```
@@ -0,0 +1,147 @@
+        update_password.password = new_password
+        s.commit()
+        return True
+    else:
+        return False
+
+
+def user_delete_by_id(user_id):
+    delete_user = s.query(User).filter_by(id=user_id).first()
+    s.delete(delete_user)
+    s.commit()
+    return True if not s.query(User).filter_by(id=user_id).all() else False
```

Ruff (0.8.2)

Use `bool(...)` instead of `True if ... else False`

Replace with `bool(...)

(SIM210)